### PR TITLE
Added title to oam_ref_schema 

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: ashish <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@Revolyssup build check failures...

File: templates/oam/workloads/v1.4.1/virtualservice.mesh.meshery.layer5io.schema.json [hunk 1]
```diff
@@ -85,5 +85,6 @@
     }
   },
   "type": "object",
+  "title":"virtualservice",
```
Comment (leecalcote) [inline templates/oam/workloads/v1.4.1/virtualservice.mesh.meshery.layer5io.schema.json:88]: Is this been dynamically generated or manually generated?

Comment (leecalcote) [reply templates/oam/workloads/v1.4.1/virtualservice.mesh.meshery.layer5io.schema.json:88]: If manual, can we set proper grammar here with capitalization and whitespace?

If manual, how are we guaranteeing new components are autogenerated properly and completely during runtime?

Comment (Revolyssup) [reply templates/oam/workloads/v1.4.1/virtualservice.mesh.meshery.layer5io.schema.json:88]: In this case these are generated manually, but in the workflows as well as runtime, these look like this:
"WasmPlugin"
"DestinationRule"
"EnvoyFilter"
"Gateway"
"ServiceEntry"
"Sidecar"
"VirtualService"
"WorkloadEntry"
"WorkloadGroup"
"AuthorizationPolicy"
"PeerAuthentication"
"RequestAuthentication"
"Telemetry"

Comment (Revolyssup) [reply templates/oam/workloads/v1.4.1/virtualservice.mesh.meshery.layer5io.schema.json:88]: So I should change this to VirtualService and so on and so on

Comment (leecalcote) [reply templates/oam/workloads/v1.4.1/virtualservice.mesh.meshery.layer5io.schema.json:88]: Actually, ideally "Virtual Service", if this is the value that we're using as a display name.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**